spec/integration/admin_items_spec.rb

```diff
@@ -6,7 +6,6 @@
 
     it "an admin can see an item listing page" do
       mock_admin
-
```

Comment:
yeah... i had some internal conflict on that. I landed on the side of `visit items_path` being a setup action. its how you set the test up to start is how i thought of it. Its not clicking on anything or filling anything in, just prepping the start of the test. I dont feel strongly on it though, so if you do let me know and i can definitely change it.